miden/src/cli/debug/executor.rs

```diff
@@ -180,23 +180,36 @@ impl DebugExecutor {
 
     /// print help message
     fn print_help() {
-        let message = "---------------------------------------------------------\n\
+        let message = "---------------------------------------------------------------------\n\
             Miden Assembly Debug CLI\n\
-            ---------------------------------------------------------\n\
-            !next        steps to the next clock cycle\n\
-            !play        executes program until completion or failure\n\
-            !play.n      executes n clock cycles\n\
-            !prev        steps to the previous clock cycle\n\
-            !rewind      rewinds program until beginning\n\
-            !rewind.n    rewinds n clock cycles\n\
-            !print       displays the complete state of the virtual machine\n\
-            !stack       displays the complete state of the stack\n\
-            !stack[i]    displays the stack element at index i\n\
-            !mem         displays the complete state of memory\n\
-            !mem[i]      displays memory at address i\n\
-            !clock       displays the current clock cycle\n\
-            !quit        quits the debugger\n\
-            !help        displays this message";
+            ---------------------------------------------------------------------\n\
+            next               steps to the next clock cycle\n\
+            next <c>           steps to the next `c` clock cycles\n\
```

Comment:
I'm not sure if `steps to the next `c` clock cycles` reads well.  Maybe we could change it to:
```
            next                  progress `1` clock cycle\n\
            next <c>           progress `c` clock cycles\n\
```

What do you think?

Reply:
I'd probably do something like:

```
next          moves to the next clock cycle\n\
next <n>      moves `n` clock cycles forward\n\
```

Reply:
Done!